.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: python
+python: 3.6
+dist: trusty
+sudo: true
+before_install:
+    - pyenv versions
+    - pyenv install --skip-existing
+    - pyenv rehash
+    - pyenv global $(cat .python-version)
+    - pip install -U pip pipenv wheel
+    - pipenv install --dev --deploy
+    - pipenv check -i '36810'
```

Comment:
Can we add a comment saying what this is?

Reply:
Added